utils/soldoutFinder.js

```diff
@@ -70,9 +85,9 @@ let generateFile = () => {
     fs.writeFileSync(`soldout_list_${d}.json`, '', (err) => {
       if (err) throw err;
     });
-    console.log('generated file');
   }
-}
+};
 
-generateList();
-// setInterval(function(){ generateList() }, 50000);
+let runInterval = (intervalLength, getSoldOut, getRestock) => {
+  setInterval(function(){ generateList(getSoldOut, getRestock) }, (intervalLength || 10000));
```

Comment:
not a big deal, but you should use `const` if the variable isn't being reassigned. I also think it's weird to switch between different notations -- why not be consistent with arrow functions?